hivemind/modules/api/models/profile.py

```diff
@@ -46,11 +64,35 @@ class Meta:
 
         ordering = ['id', 'school_id', 'last_name']
 
+    def get_full_name(self):
+        """
+        Returns the full name of the person
+        """
+        if self.preposition:
+            return f"{self.first_name} {self.preposition} {self.last_name}"
+
+        return f"{self.first_name} {self.last_name}"
+
+    def get_last_name(self):
```

Comment:
Could we turn this and `get_full_name` into a property ? It would make it easier to serialize on the API in the future.